compose.yml

```diff
@@ -1,7 +1,10 @@
 version: '1'
```

Comment:
My docker version (26.0.0) complains about `version` being obsolete. Not an error, just a warning

Reply:
Removed